arduino/WandCore/WandCore.ino

```diff
@@ -175,50 +190,135 @@ void setup()
   // Configure RGB LED
   setupRGBLED(21, 17, 16);
 
-  // Turn RGB LED off
-  setRGBLED(0, 0, 0);
-
   // Use the built-in LED as visual feedback
   pinMode(LED_BUILTIN, OUTPUT);
 
-  // Double flash to indicate end of setup
+  // Flash onboard and RGB LEDs to indicate end of setup
   digitalWrite(LED_BUILTIN, HIGH);
+  setRGBLED(255, 0, 0);
   delay(200);
   digitalWrite(LED_BUILTIN, LOW);
+  setRGBLED(0, 0, 0);
   delay(200);
   digitalWrite(LED_BUILTIN, HIGH);
+  setRGBLED(0, 0, 255);
+  delay(200);
+  digitalWrite(LED_BUILTIN, LOW);
+  setRGBLED(0, 0, 0);
+  delay(200);
+  digitalWrite(LED_BUILTIN, HIGH);
+  setRGBLED(0, 255, 0);
   delay(200);
   digitalWrite(LED_BUILTIN, LOW);
+  setRGBLED(255, 255, 255); // white for power mode, which is default
 
   // testDevices();
 }
 
-void testDevices()
+//void testDevices()
+//{
+//  // Test RGB LED
+//  setRGBLED(255, 0, 0);
+//  delay(1000);
+//  setRGBLED(0, 255, 0);
+//  delay(1000);
+//  setRGBLED(0, 0, 255);
+//  delay(1000);
+//  setRGBLED(0, 0, 0);
+//  delay(1000);
+//
+//  // Test color sensor
+//  setColorSensorLED(true);
+//  delay(1000);
+//  setColorSensorLED(false);
+//  for (int i = 0; i < 5; i++)
+//  {
+//    getRGB(&sensorR, &sensorG, &sensorB);
+//    delay(200);
+//  }
+//}

```

Comment:
Why did you comment this out? If you don't want to use it, just comment out or remove its invocation.